### ISSUE TITLE
Ui

### ISSUE THREAD

**Issue by Grizzly127** [Kompakkt/Repo#42]

**Comment by Grizzly127** [comment]
### Upload Metadata

We should also discuss how the Upload of the Metadata should look like, it is also very different: 

kompakkt: 
<img width="1040" alt="Bildschirmfoto 2024-06-12 um 15 26 01" src="https://github.com/Kompakkt/Repo/assets/44752496/be8d1527-f7e6-487f-b6f0-c82812103fe7">
sk: 
<img width="1047" alt="Bildschirmfoto 2024-06-12 um 15 25 22" src="https://github.com/Kompakkt/Repo/assets/44752496/0c4c7c66-b45f-4f6c-b684-e967ff297d0d">

persons in kompakkt: (also it is not clear what is mandatory and what not, you have to give a contact person for example, but it is not clear to the user, that this information is needed) 
kompakkt:
<img width="1013" alt="Bildschirmfoto 2024-06-12 um 15 28 05" src="https://github.com/Kompakkt/Repo/assets/44752496/e11cfb0d-c28a-46c8-bdc2-7cb00c592399">
sk: 
<img width="1034" alt="Bildschirmfoto 2024-06-12 um 15 30 54" src="https://github.com/Kompakkt/Repo/assets/44752496/251bb2ef-e2b2-40d1-90d4-fe9e7d3211b7">

**Comment by lozanaross** [comment]
> ### Metadata next to the object
> I noticed that the information for the object view are very different in kompakkt and sk. In terms of what kind of information you can see and especially the “Persons and Institutions”/“Related agents” section. This issue is to discuss and decide which information is important here and how we decide to make it consistent in kompakkt and sk?
> 

Thanks Senya, this is a good issue to resolve. I think we should aim to align this part of the presentation thus:

- [ ] We need to order Licenses, Persons and Institutions & External Links in SK in the order as they are in Kompakkt & then we can add all other SK data (like Creation) below. 
- [ ] We need to rename Related agents in SK to 'Related persons and institutions'
- [ ] We need to rename Persons and institutions in Kompakkt to 'Related persons and institutions'
- [ ] When the user opens the dropdown for that field, the should see in both apps the roles for e.g. Rightsowner / Creator styled in the same way - I propose we use the SK styling for that as it's a bit more logical, the pill design doesn't make sense as these elements are not buttons to be clicked. The difference would be though that in SK the names are also hyperlinked because they are links to WB, in Kompakkt the names will not be links. In addition, we should add the address like so "Contact address: xyz@xyz.org" (for people) and "Contact address: Germany, 10785 Berlin, Stauffenbergstr. 41-42" (all in one line, independent of how the order is entered by users in WB vs Kompakkt). If users don't add an address in WB, this will simply be invisible in SK. 

Can you make two new mockups based on the above and paste one here for Kompakkt changes & one also in SK / Gitlab issue for the SK changes that need to be made there.

**Comment by lozanaross** [comment]
> ### Upload Metadata
> We should also discuss how the Upload of the Metadata should look like, it is also very different:

Thanks for this comment @Grizzly127 - I think we should vote which option we like better - a vertical or horizontal wizard and align both to the same style. @HeyItsBATMAN @ZetOE - what's your opinion here?

I personally I like the "horizontal" style of the SK version a bit better (the other long scroll thing somehow seems endless and might put users off of exploring all options), but perhaps it's also a matter of making the data a bit more lean for Kompakkt (I think in SK we already reduced it a lot, so nothing to cut here, but perhaps we can also add some new bits here especially when it comes to possibility to add new people, or new things in the database if they are not already findable with the search option). If there is less data entry fields, then maybe the long scroll is also not so bad. 

By making the data more lean, I mean for example all these fields under General Information: 

Statement
Object type
Tags
Discipline

We don't have these in SK, and I wonder what's their function and how many people use them? @HeyItsBATMAN @ZetOE do you have any remarks here - I feel like I've never seen tags or discipline being used as navigation / exploration filters, so what are they for? 

If we go through all metadata fields one by one and identify if some can be cut, then perhaps you can try a redesigned mockup @Grizzly127 that puts Kompakkt's structure in something that looks more like SK? Or vice versa. If the Kompakkt's structure is simplified, we can try to put SK's wizard into the long form scroll.

**Comment by HeyItsBATMAN** [comment]
> I think we should vote which option we like better - a vertical or horizontal wizard and align both to the same style. @HeyItsBATMAN @ZetOE - what's your opinion here?

I'm strongly in favor of the SK style. The long scroll of Kompakkt really complicates things, and also makes it unnecessarily difficult to see what's missing.

> Statement
Object type
Tags
Discipline

@ZetOE can probably tell more about the purposes of each of those. But I think the design of SK can be applied to Kompakkt even without removing fields.

**Comment by Grizzly127** [comment]
### Update Metadata next to the object 

The order for the information next to the object in kompakkt is now this:
License - Related persons and institutions - External links
<img width="625" alt="Bildschirmfoto 2024-06-18 um 15 20 18" src="https://github.com/Kompakkt/Repo/assets/44752496/bec85c4c-2b44-4a77-be6a-fc793ed742cc">

Can we align the information options with the title above and use a 32px padding right and left of the information card? 

In kompakkt we need to change the license information like this then it is the same like in sk: 
<img width="504" alt="Bildschirmfoto 2024-06-18 um 15 20 40" src="https://github.com/Kompakkt/Repo/assets/44752496/2e8bdf65-d1f9-4c1c-9810-7c7b28b843a8">

before I continue to make the Related persons and institutions information looking good, just one question for the Contact address information. Is there for every person/institute behind their role? (That means that it sometimes is repeated) or is there one information for who is the contact person/institution? 

one person with two roles: 
left: contact address repeated, right: role as contact person with address
<img width="675" alt="Bildschirmfoto 2024-06-18 um 15 26 00" src="https://github.com/Kompakkt/Repo/assets/44752496/77e764ba-7cd7-42c9-a268-2a2214427e24">

one person and one institution having a role: 
left: one person and one institution have a role and contact addresses, right: the institution functions as contact person
<img width="540" alt="Bildschirmfoto 2024-06-18 um 15 26 49" src="https://github.com/Kompakkt/Repo/assets/44752496/ff145a32-0beb-43eb-a972-da05cdf02ef4">

**Comment by lozanaross** [comment]
@Grizzly127 As discussed today, the cases when the same person or institution have more than one role, the roles can be listed on one line with a comma or & sign, eg: Rights owner & Creator: Lozana Rossenova ... The address info is then listed below (only once). 

Also from discussion today with @Grizzly127 & @vmalieske  - Next button will be used as is currently in SK & Kompakkt - so it only progresses to the next stage in the stepper wizard and not through individual metadata. To check the different metadata sections, users can navigate via the sidebar. The Next button will be titled "Finalize" and will be greyed out until the users complete at least the 3 mandatory fields, then it will be possible to progress to the final step in the stepper. We will also test the inclusion of a "Save" button in the bottom area of the wizard, so that users feel more secure they won't lose their metadata if they don't immediately finish the upload.

cc @HeyItsBATMAN

**Comment by Grizzly127** [comment]
**Decision for displaying the persons and institutions next to the object:** 
<img width="429" alt="Bildschirmfoto 2024-07-17 um 16 06 57" src="https://github.com/user-attachments/assets/bd5bbb8d-d3d4-47f9-9695-26dd8bb17b3a">
If a person or institution has more than one role, it can be listet with a comma. 
and under every person or institution you can find a contact address. 
Let's make a gap of 8px between name and address and 16px between the information. 
<img width="398" alt="Bildschirmfoto 2024-07-17 um 16 16 26" src="https://github.com/user-attachments/assets/ad337755-cc71-474a-bbbd-b76436b30a70">

**Comment by Grizzly127** [comment]
### Upload Metadata

This is my suggestion for the design for the upload formular: 
- Change the title "Edit object" to our H2 font, open sans, 16px, bold
- the order of the information options is adjusted to the order we defined for the information next to the object now. 
- the symbol at the option titles is the same now for every option, but the optional information gets a title to be separated from the mandatory information
- the arrow symbol for the mandatory information is in the pink color as long as it is not filled out
- the buttons on the bottom are for navigating the stepper and therefore they are disables as long as the mandatory information is not filled out. We changed the right button from "Next" to "Finalize" to clarify that it is not to guide the user through the information options
- under everything that you can fill out there ist a save button to make the user feel save to keep their metadata. 
- I also used a grid for the design, let me know if it is already important to make the adjustments that detailed, then I can give more information about the measurements @lozanaross @vmalieske 
<img width="508" alt="Bildschirmfoto 2024-07-17 um 17 05 05" src="https://github.com/user-attachments/assets/ff89f187-28ed-49e8-bcfe-3c57b7ca7d18">
<img width="501" alt="Bildschirmfoto 2024-07-17 um 17 04 25" src="https://github.com/user-attachments/assets/56cb1ec9-32cf-4f47-b555-2a5f10280a0e">
<img width="513" alt="Bildschirmfoto 2024-07-17 um 17 06 35" src="https://github.com/user-attachments/assets/dae574ab-ecd0-4676-8cd3-5934b2d9a50b">
<img width="514" alt="Bildschirmfoto 2024-07-17 um 17 06 50" src="https://github.com/user-attachments/assets/4fc964db-7909-40ca-9c49-58f3a544f14b">
<img width="506" alt="Bildschirmfoto 2024-07-17 um 17 07 05" src="https://github.com/user-attachments/assets/b952b6a1-79f2-403e-9ae2-e72549d422c7">
<img width="495" alt="Bildschirmfoto 2024-07-17 um 17 07 16" src="https://github.com/user-attachments/assets/0e92f311-4c2a-498b-9706-0776c5c69a91">
<img width="490" alt="Bildschirmfoto 2024-07-17 um 17 07 39" src="https://github.com/user-attachments/assets/8baa0561-74a9-4cba-9845-0a44265cf57e">
<img width="493" alt="Bildschirmfoto 2024-07-17 um 17 07 52" src="https://github.com/user-attachments/assets/adefa4f7-f38d-4786-87c0-3156b92233f9">

**Finalize upload**
- use the H2, open sans, 16, bold for the Metadata header
- gap between elements: 16px
- should we also add the roles that you gave the persons here? 
<img width="495" alt="Bildschirmfoto 2024-07-17 um 17 08 07" src="https://github.com/user-attachments/assets/3eb2e8d5-4907-4f54-882b-9335dbec3abf">
<img width="699" alt="Bildschirmfoto 2024-07-17 um 17 10 46" src="https://github.com/user-attachments/assets/7f9a23bf-33ba-417d-a70c-999636df7910">

@lozanaross @vmalieske 
cc @HeyItsBATMAN

**Comment by Grizzly127** [comment]
### Update for Metadata next to the object
to make it look better, we could decide to make it look more like a table, the roles have always the same space (length of 120px) and are separated through a new line, therefore the persons or institutions always start at the same position and it looks cleaner. What do you think? 
<img width="537" alt="Bildschirmfoto 2024-07-17 um 16 42 43" src="https://github.com/user-attachments/assets/c99b2ce7-2d7b-46bb-be28-9e33c1c038dc">
@lozanaross @vmalieske

**Comment by HeyItsBATMAN** [comment]
> should we also add the roles that you gave the persons here?

Sounds like a good idea to me. In general, we could include the information of all required fields here (on the finalize page), and re-use the design/component from the detail page for this?

**Comment by Grizzly127** [comment]
Good idea, then we would have it in the same way like next to the object here: 
<img width="602" alt="Bildschirmfoto 2024-07-17 um 17 37 08" src="https://github.com/user-attachments/assets/ffa944eb-28ef-429e-8e82-d56ea0b1bf56">

**Comment by lozanaross** [comment]
@Grizzly127 see my comment in Gitlab, i think the Grid presentation is a bit confusing. I think it would look better if we keep all roles on one line, separated by a comma, but maybe we move the contact name to a new line to avoid awkward gaps.

**Comment by lozanaross** [comment]
@Grizzly127 & @HeyItsBATMAN - one more question regarding the metadata upload form - are all the fields really the same between Kompakkt and SK - so icluding creation & the related properties like software, technique, etc? Perhaps it's worth thinking about a slightly alternative way to add these because in SK they are coming from Wikibase, in Kompakkt they can be entered manually, so the entry fields cannot be exactly the same as in SK.

**Comment by Grizzly127** [comment]
> @Grizzly127 & @HeyItsBATMAN - one more question regarding the metadata upload form - are all the fields really the same between Kompakkt and SK - so icluding creation & the related properties like software, technique, etc? Perhaps it's worth thinking about a slightly alternative way to add these because in SK they are coming from Wikibase, in Kompakkt they can be entered manually, so the entry fields cannot be exactly the same as in SK.

Right now, this is the Creation part for kompakkt: 
<img width="697" alt="Bildschirmfoto 2024-08-02 um 13 18 41" src="https://github.com/user-attachments/assets/e8e24fad-a095-4245-9d1d-9acc658b98c7">

and this for semantic kompakkt: 
<img width="1007" alt="Bildschirmfoto 2024-08-02 um 13 18 50" src="https://github.com/user-attachments/assets/ea3e1efe-e0f8-4605-a255-09196984ccba">

How do the fields look like if the information is coming from Wikibase? I'm not sure if I have seen it somewhere yet. 
@lozanaross

**Comment by lozanaross** [comment]
@Grizzly127 The Wikibase entries have the magnifier glass because it's only allowing search of existing entries. In Kompakkt I think you can just enter plain text? Or can you also search existing entries? Not sure, but probably you don't need the magnifier glass there. And maybe "Program" in Kompakkt can also be renamed Software to keep it more similar to SK. It also looks like creation is mandatory in Kompakkt, but it's not in SK, so that can be something further aligned. What fields are mandatory, what not, etc.

**Comment by Grizzly127** [comment]
> @Grizzly127 The Wikibase entries have the magnifier glass because it's only allowing search of existing entries. In Kompakkt I think you can just enter plain text? Or can you also search existing entries? Not sure, but probably you don't need the magnifier glass there. And maybe "Program" in Kompakkt can also be renamed Software to keep it more similar to SK. It also looks like creation is mandatory in Kompakkt, but it's not in SK, so that can be something further aligned. What fields are mandatory, what not, etc.

okay, then there will be just small differences: 
semantic kompakkt: 
- there is the magnifier glass because you can search through wikibase entries
- is the equipment field also connected to Wikibase? @lozanaross 
<img width="639" alt="Bildschirmfoto 2024-08-05 um 15 46 33" src="https://github.com/user-attachments/assets/7dfe0ba4-4d45-4b31-95f8-a43d5395dfa1">

kompakkt:
- no magnifier glass, because you can only enter plain text
- Creation is not mandatory in kompakkt (it just turned read, when you clicked on it to add something. The fields "technique and software" were mandatory to add a creation though, I would suggest not to have this two mandatory like in sk)
<img width="645" alt="Bildschirmfoto 2024-08-05 um 15 46 40" src="https://github.com/user-attachments/assets/108edcbe-cb23-4354-98b3-f2a134697c51">

**Comment by lozanaross** [comment]
@Grizzly127 

Great, thanks, to the questions below:
> * there is the magnifier glass because you can search through wikibase entries
> * is the equipment field also connected to Wikibase? @lozanaross
Equipment is added to Wikibase, but it's a free-text field, so the user can just add anything here and it's saved as plain text (not data) in Wikibase, this is a compromise because we didn't want to limit what kind of equipment users can enter. It's somewhat similar to what would happen in Kompakkt I assume.

> kompakkt:
> 
> * no magnifier glass, because you can only enter plain text
> * Creation is not mandatory in kompakkt (it just turned read, when you clicked on it to add something. The fields "technique and software" were mandatory to add a creation though, I would suggest not to have this two mandatory like in sk)

Yes, this sounds good to me. 

Additionally as discussed today, you may need to consider some additional templates that specify how users add new 'entities' in Kompakkt itself, which wouldn't exist in SK because we don't need it for WB.

**Comment by Grizzly127** [comment]
### Metadata persons and institutions kompakkt

first suggestion for the form for adding persons and institutions:
- you have one search field, where you can either look for persons/institutions that already exist. If they exist you can choose them and all the fields below are filled out automatically by the already given information. If the person/institution doesn't exist the user keeps typing in their name and fills out the information below manually. (similar to when you write an email and the program know the email address and can fill it out or you can keep typing in the address if it is a new one) 
- if you fill it out for a new person/institution, you can choose between person and institution and then choose which contact information should be given.

<img width="711" alt="Bildschirmfoto 2024-08-06 um 16 23 18" src="https://github.com/user-attachments/assets/2dc177bc-95bb-4550-b02d-f8db45570665">

<img width="571" alt="Bildschirmfoto 2024-08-06 um 16 15 47" src="https://github.com/user-attachments/assets/128b41fd-e6be-41b0-8dea-2114ff54ec86">

<img width="664" alt="Bildschirmfoto 2024-08-06 um 16 18 49" src="https://github.com/user-attachments/assets/5b1c63d5-10e3-43d9-ab5a-84d764e540d2">

@lozanaross

**Comment by Grizzly127** [comment]
### Related physical objects

Let's discuss if we need this option or if we keep it minimal.
For example the user can only type in a name of an object and adds an external link to more information and resources

<img width="596" alt="Bildschirmfoto 2024-08-06 um 16 29 49" src="https://github.com/user-attachments/assets/f84ca1e2-ddc2-4b92-af97-a43b6bc70db8">

@lozanaross @HeyItsBATMAN @ZetOE

**Comment by Grizzly127** [comment]
### Metadata Persons and Institutions

We realized that sometimes it is not clear to the user that it is mandatory to add Rightowner and Creator. 
I am working on it to make it more clear. This is the first idea: 
The two roles are mentioned, but in red, as long as no person or institution is allocated. 
As soon as both are allocated, the Save button becomes usable. Other roles appear when they are added. 
<img width="541" alt="Bildschirmfoto 2024-08-22 um 12 30 37" src="https://github.com/user-attachments/assets/fcca59f8-35aa-4cad-9a78-76e2a947bf9e">
<img width="545" alt="Bildschirmfoto 2024-08-22 um 12 30 44" src="https://github.com/user-attachments/assets/d315d0ac-856b-4fd7-b0bf-021da057a85e">
<img width="487" alt="Bildschirmfoto 2024-08-22 um 12 31 27" src="https://github.com/user-attachments/assets/71debf12-d444-495f-8f28-658c1299fe3f">

@lozanaross @ZetOE

**Comment by lozanaross** [comment]
@Grizzly127 comments below:

> ### Metadata persons and institutions kompakkt
> first suggestion for the form for adding persons and institutions:
> 
> * you have one search field, where you can either look for persons/institutions that already exist. If they exist you can choose them and all the fields below are filled out automatically by the already given information. If the person/institution doesn't exist the user keeps typing in their name and fills out the information below manually. (similar to when you write an email and the program know the email address and can fill it out or you can keep typing in the address if it is a new one)

Not sure if this can actually work with out Angular framework - @vmalieske can you confirm? If yes, great. 

> * if you fill it out for a new person/institution, you can choose between person and institution and then choose which contact information should be given.

Question - will the form dynamically adjust if a person is chosen vs institution? E.g. if a person is chosen we only need Email address & Phone; if instittuion - maybe only physical address? If the form stays exactly the same, do we really need to make a distinction between person / institution? We could also get rid of this extra tick box if nothing else is determined by it. 

Additionally, let's adjust the text to: "**E**mail address" and "Phone number" (with spaces). Can we get rid of "Note" - why do we need it? Let's reduce complexity as much as possible. 

The rest of the form looks good though.

**Comment by lozanaross** [comment]
@Grizzly127 comments below:

> ### Related physical objects
> Let's discuss if we need this option or if we keep it minimal. For example the user can only type in a name of an object and adds an external link to more information and resources

I am in favor of keeping this more minimal - what do  you think @HeyItsBATMAN @ZetOE? For me just a title of the object, link to Wikidata or another external link would be sufficient. 

I think the link text should be more description, e.g.:
"Add link to external source information about the physical object, e.g. Wikidata, or institutional website"

Users should be able to add more than one link. 

What other information is otherwise added via Kompakkt? @Grizzly127 can you provide a bullet list below and we can discuss line by line if we can get rid of it all, or we keep a few. It's important to check how much is this feature actually used in the current version of Kompakkt before replicating all of it, I think.

**Comment by Grizzly127** [comment]
> What other information is otherwise added via Kompakkt? @Grizzly127 can you provide a bullet list below and we can discuss line by line if we can get rid of it all, or we keep a few. It's important to check how much is this feature actually used in the current version of Kompakkt before replicating all of it, I think.

If the user adds a related physical object the following information can be filled out: 

- General Information with Title, Description and Collection: 
<img width="940" alt="Bildschirmfoto 2024-10-01 um 14 33 51" src="https://github.com/user-attachments/assets/b6357573-ab2d-48eb-b18a-11334116d064">

- Place: 
<img width="879" alt="Bildschirmfoto 2024-10-01 um 14 28 28" src="https://github.com/user-attachments/assets/a03835b1-0308-4955-918d-6394aabd9a34">

- a new person can be added: 
<img width="963" alt="Bildschirmfoto 2024-10-01 um 14 29 06" src="https://github.com/user-attachments/assets/08361598-be5e-4a99-aa9f-5f7f15776fe1">

- same with an institution: 
<img width="916" alt="Bildschirmfoto 2024-10-01 um 14 29 51" src="https://github.com/user-attachments/assets/e91d3e78-d2cd-4126-bcb7-c4bdf27d8f50">

- External identifiers with Type and Value: 
<img width="484" alt="Bildschirmfoto 2024-10-01 um 14 31 11" src="https://github.com/user-attachments/assets/b6426362-a6a9-4183-8238-01d49f367529">

- External Links with Description and Value: 
<img width="484" alt="Bildschirmfoto 2024-10-01 um 14 30 52" src="https://github.com/user-attachments/assets/c903e26f-5a31-46ef-9335-b7d50b209f4b">

- Bibliographic References with Value: 
<img width="522" alt="Bildschirmfoto 2024-10-01 um 14 32 12" src="https://github.com/user-attachments/assets/25a0df02-3a41-485d-a99a-866104d339ed">

- Other with description and Value: 
<img width="490" alt="Bildschirmfoto 2024-10-01 um 14 32 31" src="https://github.com/user-attachments/assets/c71c6481-3d95-4efc-8432-1a73074b30ed">

- Metadata files: 
<img width="561" alt="Bildschirmfoto 2024-10-01 um 14 33 06" src="https://github.com/user-attachments/assets/c96e3599-6bcf-47e4-89e1-71701e318bf4">

@lozanaross @ZetOE

**Comment by vmalieske** [comment]
### Persons and Institutions

Unfortunately, I haven't got as far as I wanted to, but this is my current state:

1. You can search for existing persons and institutions at the same time, and the corresponding values are entered in the form. There are no fields for prename and name in the mockup. They are not really necessary if you add existing persons/institutions, but maybe it is irritating if you have to write the name to add in the input field for the search? What do you think?

![image](https://github.com/user-attachments/assets/639f7af2-5cb7-407c-84a2-338f70bc119b)
![image](https://github.com/user-attachments/assets/7d040322-bd64-40d5-8b92-52f208dc3618)

3. The checkboxes for persons and institutions are used to indicate which fields are required, depending on whether they are persons or institutions. When a person or institution is selected, the corresponding checkbox is automatically activated.

![image](https://github.com/user-attachments/assets/de41d937-28fd-4bac-acc1-cddf5bd3c8bd)
![image](https://github.com/user-attachments/assets/6f06cc16-1340-49a2-8d67-69db184cf153)

@lozanaross @Grizzly127 @ZetOE

**Comment by vmalieske** [comment]
One question about the creation-data when adding more than one dataset.
In semantic kompakkt it looks like that:
![image](https://github.com/user-attachments/assets/1f939e9e-545c-4f96-bd7c-d3183c8d3c51)

Every field has its own card. Should it be that way or should every field be in one card like that:
![image](https://github.com/user-attachments/assets/3e1c582f-dde8-43ec-bcfd-8dec219112c6)

@lozanaross @HeyItsBATMAN @Grizzly127 @ZetOE

**Comment by HeyItsBATMAN** [comment]
Creation date can only be one.
I think separate cards make more sense here because they can then be deleted one by one.
But then on the Entity detail page, they can probably be grouped together?

**Comment by Grizzly127** [comment]
> 1. You can search for existing persons and institutions at the same time, and the corresponding values are entered in the form. There are no fields for prename and name in the mockup. They are not really necessary if you add existing persons/institutions, but maybe it is irritating if you have to write the name to add in the input field for the search? What do you think?

I think it can work like this for the user, it works like this on other platforms as well. And we reduce the number of input fields. If we get the feedback that it is confusing, we can still change it. 

> 3. The checkboxes for persons and institutions are used to indicate which fields are required, depending on whether they are persons or institutions. When a person or institution is selected, the corresponding checkbox is automatically activated.

And if it is a new person, the user can choose if it is a person or institution, right? 

Three small things: 
- on top it says that there needs to be a person as rights owner and contact person, but it is rights owner and creator, isn't it? 
- Rights owner not written together for the tick-boxes.
- There is a mistake in the last sentence: You muSt (nur ein S) specify at least one agent for each of theSe (S statt R) roles. 

@lozanaross @ZetOE  @vmalieske

**Comment by Grizzly127** [comment]
### Corrections for current metadata form Part 1

**General Information** 
- For me it's not clear what a statement is
- What is meant with object type? are there examples for what an object type is? 
- Tags often appear two or three times in the dropdown: 
<img width="410" alt="Bildschirmfoto 2024-12-17 um 13 22 08" src="https://github.com/user-attachments/assets/ad3092b8-537c-48a3-8a40-227566c4f20b" />

- I would say "Separate tags with commas or by pressing Enter/Return." and "Separate disciplines with commas or by pressing Enter/Return."
<img width="999" alt="Bildschirmfoto 2024-12-17 um 13 14 52" src="https://github.com/user-attachments/assets/9e925b56-10ee-419e-bb39-214322e07696" />


**Licence**
- I think the licence part is okay, maybe we can add a user information here as well like "Select a license that suits your object best." or that fits the way other users are allowed to use it. 

**Persons**
   **General Information:** 
- we could already call it "Related persons" here
- on the first look it looks like there is no General Information, maybe we can leave this out and only display the name. 
- "You are adding onto an existing person" and the name needs to be differentiated. It looks the same, but one is a user information and the other is the persons name, I think it needs to look different.
- The sentence "You are adding onto an existing person" is in general a bit confusing. I know that it is meant that the user is able to add new information to a person and then the new information is also saved at the data base connected to the person. But I think the user doesn't understand this concept here. Maybe we just leave this out, the user can see in the form that there is already an email address for example. This information is only interesting in the Contact section and there we have another user information. 
<img width="955" alt="Bildschirmfoto 2024-12-17 um 13 25 47" src="https://github.com/user-attachments/assets/415fec0d-76a6-49b1-84e0-514b87de8177" />

   **Role Selection:** 
- Better only say: "Select at least one role" 
- Right now it is not clear that the user needs to select one Rights owner and one Creator in the end. The text in the beginning says one Rights owner and one Contact Person, which is not correct. The Persons section stays read if no Rights owner or no Creator is selected, but apart from that there is no information about why the form is not complete. The institution part becomes blue no matter what roles are selected. 
- In the end there needs to be one Rights owner and one Creator, no matter if it is a person or institution, right? 
- maybe we can already integrate the part from the new form below the persons and institutions part (second picture)? 
<img width="937" alt="Bildschirmfoto 2024-12-17 um 14 09 36" src="https://github.com/user-attachments/assets/953ddaa4-0f15-445b-94fa-73661c224935" />
<img width="385" alt="Bildschirmfoto 2024-12-19 um 11 06 27" src="https://github.com/user-attachments/assets/0b96a7ca-dbe0-40d8-9632-7ffaf26590b2" />

**Contact**
- "You can choose an existing contact reference or attach a new one." This is the concept I talked about before. I'm not sure if it is a common concept for the user. Maybe we can just have the fields filled out where there is already information and if there is more than one e-mail address for example, it can be shown with a dropdown. 
- If "New E-Mail address" is selected, the field is empty and ready to type something in. 
- Phone number (separated)
- For what is the notes field? Can we leave this out? 
<img width="933" alt="Bildschirmfoto 2024-12-17 um 14 10 07" src="https://github.com/user-attachments/assets/a28356a1-632c-4754-9467-76756b63b23b" />
<img width="876" alt="Bildschirmfoto 2024-12-17 um 14 10 21" src="https://github.com/user-attachments/assets/1a26dd3f-0d2c-4ee1-96cb-24b58b88f1ac" />

<img width="673" alt="Bildschirmfoto 2024-12-19 um 12 37 40" src="https://github.com/user-attachments/assets/ebe66a09-c3a4-497c-8cf4-ff76482cae2f" />
<img width="516" alt="Bildschirmfoto 2024-12-19 um 12 41 14" src="https://github.com/user-attachments/assets/b91836cd-7d47-4679-a294-f6fca5851bc6" />

**Institutions**
- same notes like in the persons field (look above) 
- What is "additional"? Better Additional information? Additional notes?
<img width="974" alt="Bildschirmfoto 2024-12-17 um 14 16 10" src="https://github.com/user-attachments/assets/de651bb8-4ed7-467c-992b-7b7393abbc85" />

@lozanaross @vmalieske @ZetOE

**Comment by Grizzly127** [comment]
### Corrections for current metadata form Part 2

**Dimensions, Creation and External Identifiers**
- I wondered if we need the + button or if the fields should always be open immediately
- Here the required fields are grey and not red with a star like in other parts of the form (e.g. Creation). I think it's because all fields are required here, but we should keep it consistent. 
- I wondered if we should have a "Add" button here? Also to have the feeling of saving the inputs for the user.
<img width="467" alt="Bildschirmfoto 2024-12-19 um 13 07 02" src="https://github.com/user-attachments/assets/b6a96275-2020-4e36-b9df-c47f3ff07162" />

- Is "Software" the better word for "Program"?
<img width="518" alt="Bildschirmfoto 2024-12-17 um 14 38 46" src="https://github.com/user-attachments/assets/e5e42096-227f-4d08-a25f-fe6672c348e3" />

**External Links**
- The Description field is here the "Link label", maybe it is not required and the user can choose if he wants to display the URL or and display text as a link. 
- The Value field should be more like: "Please enter a valid URL" and then it needs to be checked if it is a valid URL
- Mark required fields consistently
<img width="549" alt="Bildschirmfoto 2024-12-17 um 14 52 02" src="https://github.com/user-attachments/assets/3ecd9c34-6d60-4cd7-86d8-17cc9aad63a4" />

**Bibliographic References**
- "Value" is also weird here. Maybe the field can also be called "Bibliographic reference"
- Mark required fields consistently
<img width="526" alt="Bildschirmfoto 2024-12-17 um 14 55 34" src="https://github.com/user-attachments/assets/1f7f6171-237f-4ec5-b821-4c89433d862c" />

**Other**
- We need to find another term here, "additional information"/"additional notes"? 
- Does Description and Value fit here? Or more an empty text field for notes? 
- Mark required fields consistently
<img width="509" alt="Bildschirmfoto 2024-12-17 um 14 58 52" src="https://github.com/user-attachments/assets/ce73884a-4939-46e3-8ef9-9b4b21c0d762" />

**Metadate Files**
- Gap between the description sentences is too big
- the data type in the middle looks weird. We can delete it, since it is already behind the name or we need to add headings like in the upload part
- the delete button is not in the middle 
<img width="578" alt="Bildschirmfoto 2024-12-17 um 15 04 21" src="https://github.com/user-attachments/assets/aecb7ebd-2c40-477a-91b1-65374a7e9faf" />
<img width="964" alt="Bildschirmfoto 2024-12-17 um 15 04 47" src="https://github.com/user-attachments/assets/565cb95a-d5a5-4898-9a76-fddfb25d3fa9" />

**Notes:** 
- Gap between "Bibliographic references" and the number is too big
<img width="406" alt="Bildschirmfoto 2024-12-17 um 15 10 24" src="https://github.com/user-attachments/assets/e699f0ef-d413-4645-966b-3724ced80345" />

@vmalieske @lozanaross @ZetOE

**Comment by Grizzly127** [comment]
### Corrections for current metadata form Part 3

**Physical objects**
**General Information:**
- keep required fields consistent
- What is meant by Collection? Real world collection the object is from? 
<img width="984" alt="Bildschirmfoto 2024-12-17 um 15 11 36" src="https://github.com/user-attachments/assets/20273272-bede-4b1d-bb57-5a505dd56f3f" />

**Place:**
- Is "Place" the right word? Is "Location" better?
- keep required fields consistent, "Name" is now red here, but the other fields also have a star and are not red. 
- The text says that the user needs to fill in name OR geopolitical area OR a specific address, but all fields are with a star, that is confusing 
- What is a geopolitical area, is it a use case to use this field? 
<img width="932" alt="Bildschirmfoto 2024-12-17 um 15 32 07" src="https://github.com/user-attachments/assets/63028df0-2a86-48f6-bc14-2f5542f61c15" />

**Persons**
- See notes for (related) persons before
- For what are the roles here? Are they connected to the physical object then? Is Data Creator fitting for a physical object? 
<img width="933" alt="Bildschirmfoto 2024-12-17 um 15 39 55" src="https://github.com/user-attachments/assets/a776107e-44b6-4f5b-8444-e90ca51ed2e7" />
<img width="845" alt="Bildschirmfoto 2024-12-17 um 15 41 57" src="https://github.com/user-attachments/assets/1f9be3ae-d823-4e0c-965d-171422d98e78" />

Notes: 
- here the gap is also too big: 
<img width="976" alt="Bildschirmfoto 2024-12-17 um 15 44 58" src="https://github.com/user-attachments/assets/a586bee3-8930-42a8-b0fb-0c67476cd19a" />


@vmalieske @lozanaross @ZetOE
